orion-ui/src/pages/FlowRun.vue

```diff
@@ -75,16 +77,18 @@
   const api = useWorkspaceApi()
   const flowRunDetailsSubscription = useSubscription(api.flowRuns.getFlowRun, [flowRunId], { interval: 30000 })
   const flowRun = computed(() => flowRunDetailsSubscription.response)
+  const deploymentId = computed(() => flowRun.value?.deploymentId)
+  const deployment = useDeployment(deploymentId)
 
   watch(flowRunId, (oldFlowRunId, newFlowRunId) => {
     if (oldFlowRunId !== newFlowRunId) {
       selectedTab.value = 'Logs'
     }
   })
 
-  const parameters = computed(() => {
-    return flowRun.value?.parameters ? JSON.stringify(flowRun.value.parameters, undefined, 2) : '{}'
-  })
+  const flowRunParameters = computed(() => flowRun.value?.parameters ?? {})
+  const deploymentParameters = computed(() => deployment.value?.parameterOpenApiSchema ?? {})
+  const parameters = computed(() => getSchemaValuesWithDefaultsJson(flowRunParameters.value, deploymentParameters.value))
```

Comment:
schema is more accurate for this const
```suggestion
  const deploymentSchema = computed(() => deployment.value?.parameterOpenApiSchema ?? {})
  const parameters = computed(() => getSchemaValuesWithDefaultsJson(flowRunParameters.value, deploymentSchema.value))
```